psi4/driver/driver.py

```diff
@@ -1274,12 +1305,25 @@ def hessian(name, **kwargs):
             core.print_out("""hessian() switching to finite difference by gradients for partial Hessian calculation.\n""")
             dertype = 1
 
+    # At stationary point?
+    G0 = gradient(lowername, molecule=molecule, **kwargs)
+    nonzero_gradient = G0.rms() > 1.e-2  # pulled out of a hat
+    translations_projection_sound = not core.get_option('SCF', 'EXTERN') and \
```

Comment:
I realize it's pulled out of a hat, but it seems a little loose from my (very limited) intuition.  I'll do some testing and post back here soon...